modules/oe_webtools_media/src/Plugin/Validation/Constraint/WebtoolsMediaConstraint.php

```diff
@@ -31,6 +31,27 @@ class WebtoolsMediaConstraint extends Constraint {
    */
   public $blacklistMessage = 'This service is supported by a dedicated asset type or feature, please use that instead.';
 
+  /**
+   * The incorrect wcloud url message.
+   *
+   * @var string
+   */
+  public $incorrectUrlMessage = 'The provided wcloud url is not valid.';
```

Comment:
```suggestion
  public $incorrectUrlMessage = 'The provided wcloud URL is not valid.';
```
URL should be always capitalised.